### ISSUE TITLE
Update continuous integration server

### ISSUE THREAD

**Issue by aecio** [crawler-commons/crawler-commons#325]
`travis.org` has stopped building projects in favor of `travis.com`.
```
Since June 15th, 2021, the building on travis-ci.org is ceased. Please use travis-ci.com from now on. 
```

We should either migrate to `travis.com` or maybe GitHub actions.

**Comment by Chaiavi** [comment]
Does pull request #333 solve this ?

**Comment by sebastian-nagel** [comment]
Resolved by #333. Thanks everybody!
